packages/frontend/src/routes.tsx

```diff
@@ -82,6 +83,17 @@ export default createRoutesFromElements(
 
     <Route path={`${URLS.EDITOR}/*`} element={<EditorRoutes />} />
 
+    <Route
+      path={`${URLS.TILES}/*`}
+      element={
+        <Layout>
+          <Suspense fallback={<></>}>
+            <TilesRoutes />
```

Comment:
nit: would recommend Suspend within the tile page component instead; this way we can use stuff like useBackgroundQuery